docs/docs/constraints.md

```diff
@@ -4,31 +4,31 @@ title: Constraints
 
 # Constraints
 
-Constraints control where apps run to allow optimizing for fault tolerance or locality. They are made up of three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
+Constraints control where apps run to enable you to optimize for fault tolerance or location. Constraints have three parts: a field name, an operator, and an optional parameter. The field can be the slave hostname or any Mesos slave attribute.
 
 ## Fields
 
 ### Hostname field
 
-`hostname` field matches the slave hostnames, see `UNIQUE operator` for usage example.
+The `hostname` field matches the slave hostnames. See `UNIQUE operator`, below, for a usage example.
 
-`hostname` field supports all operators of Marathon.
+The `hostname` field supports all Marathon operators.
 
 ### Attribute field
 
-If the field name is none of the above, it will be treated as a Mesos slave attribute. Mesos slave attribute is a way to tag a slave node, see `mesos-slave --help` to learn how to set the attributes.
+If the field name is not the slave hostname, it will be treated as a Mesos slave attribute. A Mesos slave attribute tags a slave node, see `mesos-slave --help` to learn how to set the attributes.
```

Comment:
You can attach key/value pairs (attributes) to slaves. You can match on those attributes.